data/pid_data.py

```diff
@@ -1148,6 +1206,16 @@
        'notes': 'Controls if the fixture will treat key seven on the MCX as a lamp power-off button.',
        'value': 0x8002},
 
+      {'get_request': {'items': []},
+       'get_response': {'items': [{'name': 'enabled', 'type': 'uint16'}]},
+       'get_sub_device_range': 0,
+       'name': 'FIXTURE_ID',
+       'set_request': {'items': [{'name': 'enabled', 'type': 'uint16'}]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'notes': 'Controls if the fixture will treat key seven on the MCX as a lamp power-off button.',
```

Comment:
Should be ok now..

Mvg,

Bart van Stiphout

Mobile NL: +31648560121
Skype: bartvstiphout

www.bartvanstiphout.com <http://www.bartvanstiphout.com/>


	


Now available in the App Store:
Fixture Tester for iPhone/iPad/iPod Touch!
Find out more! <http://itunes.apple.com/us/app/fixture-tester/id368100165?mt=8>
> On 09 May 2015, at 02:01, Simon Newton <notifications@github.com> wrote:
> 
> In data/pid_data.py <https://github.com/OpenLightingProject/rdm-app/pull/20#discussion_r29986075>:
> 
> > @@ -1148,6 +1206,16 @@
> >         'notes': 'Controls if the fixture will treat key seven on the MCX as a lamp power-off button.',
> >         'value': 0x8002},
> >  
> > +      {'get_request': {'items': []},
> > +       'get_response': {'items': [{'name': 'enabled', 'type': 'uint16'}]},
> > +       'get_sub_device_range': 0,
> > +       'name': 'FIXTURE_ID',
> > +       'set_request': {'items': [{'name': 'enabled', 'type': 'uint16'}]},
> > +       'set_response': {'items': []},
> > +       'set_sub_device_range': 0,
> > +       'link': '',
> > +       'notes': 'Controls if the fixture will treat key seven on the MCX as a lamp power-off button.',
> This notes field looks like it was copied from the PID above and doesn't match the PID name.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/OpenLightingProject/rdm-app/pull/20/files#r29986075>.
>